internal/kafka/internal/workers/kafka_mgrs/kafkas_mgr.go

```diff
@@ -377,6 +393,55 @@ func (k *KafkaManager) updateClusterStatusCapacityMaxMetric(c services.KafkaStre
 	metrics.UpdateClusterStatusCapacityMaxCount(c.CloudProvider, c.Region, c.InstanceType, c.ClusterId, float64(c.MaxUnits))
 }
 
+// TODO: this is a temporary reconciler added to value the `actual_kafka_billing_model` and `desired_kafka_billing_model` for
+// kafkas where billing_model has always been empty. See https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/1115#discussion_r902415419 for details
+func (k *KafkaManager) reconcileBillingModel(kafkas dbapi.KafkaList) serviceErr.ErrorList {
```

Comment:
Can we add a more  migration specific name to this method? `reconcileBillingModel` is very generic

Reply:
Any suggestion?

Reply:
for example tempMigrateEmptyBillingModels